pkg/operator/ceph/operator.go

```diff
@@ -122,31 +127,28 @@ func (o *Operator) updateOperatorLogLevel() error {
 // Run the operator instance
 func (o *Operator) Run() error {
 
-	if o.operatorNamespace == "" {
+	if o.config.OperatorNamespace == "" {
 		return errors.Errorf("rook operator namespace is not provided. expose it via downward API in the rook operator manifest file using environment variable %q", k8sutil.PodNamespaceEnvVar)
 	}
 
+	// TODO: needs a callback?
```

Comment:
Why would a callback be needed?

Reply:
Because this does not reload if the `rook-ceph-operator-config` CM is updated.

Reply:
FYI: I'm fixing this in an upcoming refactor PR

Reply:
Actually fixed in `removeOldAdmissionControllerDeployment()`